ci-operator/step-registry/assisted/common/verify/assisted-common-verify-commands.sh

```diff
@@ -36,6 +36,11 @@ esac
 export KUBECONFIG=${SHARED_DIR}/kubeconfig
 mkdir -p ${ARTIFACT_DIR}/reports
 
-openshift-tests run "openshift/conformance/parallel" --dry-run | \
+podman run --rm -i \
```

Comment:
it would be great to make it a bit more readable (by using an alias or a function)